recipe/test_gen_patch_json.py

```diff
@@ -27,3 +27,65 @@ def test_gen_patch_instructions():
         'c': {'addthis': 'yes'}}
 
     assert inst['remove'] == list(REMOVALS['osx-64'])
+
+
+def test_add_python_abi():
+    conditions = {
+        'python >=2.7,<2.8.0a0': 'cp27mu',
```

Comment:
@mattip, a package with the dependency given as `python >=2.7,<2.8.0a0` gets tagged with the abi tag `cp27mu`.